jujugui/static/gui/src/app/components/deployment-flow/expert-intro/expert-intro.js

```diff
@@ -105,34 +106,14 @@ class DeploymentExpertIntro extends React.Component {
                 title="Juju expert partners">
                 <div className="deployment-expert-intro__plan-details">
                   <h3>
-                    From $3577 per month
+                    {price}
                   </h3>
                   <div className="deployment-expert-intro__plan-description">
                     Default plan with standard support
```

Comment:
Should this still be here? Looks like it should be removed...

Reply:
Nope this stays as the price that is supplied by the expert here is using the default plan with standard support. Modifications to that price are further down in the flow.

Reply:
Oh I see!